app/_hub/datadome/kong-plugin-datadome/_index.md

```diff
@@ -3,4 +3,62 @@ The DataDome plugin is developed in Lua and integrates smoothly with Kong.
 This plugin will analyse in real time any API requests received. For each request, an api call is sent to DataDome Bot & Fraud Protection Platform to validate if this is legitimate or not.
 The only thing to configure is your DataDome Server Side Key.
 
-[This integration guide](https://docs.datadome.co/docs/kong-plugin) describes how to integrate DataDome with Kong API Gateway.
\ No newline at end of file
+## How it works
+
+The DataDome plugin will hook into the request/response of an API request and it is executed for every request from a client and before it is being proxied to the upstream on the access phase (see details [here](https://openresty-reference.readthedocs.io/en/latest/Directives/)).
+
+## How to install
+
+Custom plugins can be installed via LuaRocks. A Lua plugin is distributed in `.rock` format, which is
+a self-contained package that can be installed locally or from a remote server.
+
+If you used one of the official {{site.base_gateway}} installation packages, the LuaRocks utility
+should already be installed in your system.
+Install the `.rock` in your LuaRocks tree, that is, the directory in which LuaRocks
+installs Lua modules.
+
+1. Install the DataDome plugin:
+
+    ```sh
+    luarocks install kong-plugin-datadome
+    ```
+
+2. Update your loaded plugins list in {{site.base_gateway}}.
+
+    In your `kong.conf`, append `datadome` to the `plugins` field. Make sure the field is not commented out.
+
+    ```yaml
+    plugins = bundled,datadome              # Comma-separated list of plugins this node
+                                            # should load. By default, only plugins
+                                            # bundled in official distributions are
+                                            # loaded via the `bundled` keyword.
+    ```
+
+3. Restart {{site.base_gateway}}:
+
+    ```sh
+    kong restart
+    ```
+
+## Using the plugin
+
+First of all, make sure you have your DataDome Server Side Key. It is available inside your [Dashboard > Integrations](https://app.datadome.co/dashboard/management/integrations).
+
+#### Kong Gateway (OSS)
+If you already have an API configured, execute the command below replacing the following `<values>`:
+
+```shell
+curl -i -X POST http://localhost:8001/services/<YOUR_API>/plugins \
+     -F "name=datadome" \
+     -F "config.datadome_server_side_key=<server_side_key>"
+ ```
+
+#### Kong Konnect
+
+- Depending where you want to enable DataDome, select *Plugins*
+- Click *+ New Plugin*
+- On Custom Plugins, select *DataDome*
+- Fill *Datadome Server Side Key*
+- *Save*
```

Comment:
```suggestion
- Depending on where you want to enable DataDome, select `Plugins`
- Click on `+ New Plugin`
- On `Custom Plugins`, select `DataDome`
- Fill the *DataDome Server-side Key* field
- Hit `Save`
```

Reply:
Not sure where the `Datadome Server Side Key` label comes from, but it should be updated too.

Reply:
It comes from kong konnect: it seems that the platform gets the parameters (defined on the schema.lua file) like datadome-server-side-key, excludes the '-' dashes and put all the first letters capitalised, resulting `Datadome Server Side Key`.
Thats the reason.

The official doc is here:
https://dash.readme.com/project/datadome/v6.0/docs/kong-plugin

Reply:
<img width="389" alt="image" src="https://github.com/laurodd/docs.konghq.com/assets/111757181/7e4d0a13-f4fb-4f18-9aa3-4dd958a14c5d">